README.md

```diff
@@ -1,4 +1,4 @@
-# NodeSecure Scanner
+<img align="center" alt="# Nodesecure Scanner" src="https://user-images.githubusercontent.com/4438263/216045720-779bf16d-1d35-409f-a0e6-4019bda8edde.jpg" alt="@nodesecure/rc">
```

Comment:
You need to use scanner banner (not rc banner): https://user-images.githubusercontent.com/4438263/226018084-113c49e6-6c69-4baa-8f84-87e6d695be6d.jpg

Reply:
ok I have changed the image link now. Thanks for the feedback